compiler/src/dotty/tools/dotc/transform/localopt/StringInterpolatorOpt.scala

```diff
@@ -105,13 +105,25 @@ class StringInterpolatorOpt extends MiniPhase:
         lintToString(elem)
         concat(elem)
         val str = stri.next()
-        if !str.const.stringValue.isEmpty then concat(str)
+        if !str.const.stringValue.isEmpty then
+          concat(str)
```

Comment:
This is due to @sjrd asking please don't put side effects to the right. Now I need it more left-adjusted, "front and center".